app/src/main/java/com/spascoding/taskycourse/feature_auth/presentation/util/NameInitialsExtractor.kt

```diff
@@ -0,0 +1,33 @@
+package com.spascoding.taskycourse.feature_auth.presentation.util
+
+class NameInitialsExtractor(val name: String) {
+
+    fun extract(): String {
+        return when(wordCount(name)) {
+            0 -> throw IllegalArgumentException("Name must contains minimum two letters")
+            1 -> extractFromOneWord()
+            else -> extractFromMoreThanTwoWords()
```

Comment:
Little wording issue here, since your function sounds like it's used for more than two words which would exclude two. Instead you seem to want `extractFromMoreThanOneWord()`